Tests/Engine/ModuleHelp.Tests.ps1

```diff
@@ -214,15 +214,19 @@ Get-Module $ModuleName | Remove-Module
 Import-Module $ModuleName -RequiredVersion $RequiredVersion -ErrorAction Stop
 $ms = $null
 $commands =$null
-$paramBlackList = @()
+$paramBlackList = @(
+	'AttachAndDebug' # Reason: When building with DEGUG configuration, an additional parameter 'AttachAndDebug' will be added to Invoke-ScriptAnalyzer and Invoke-Formatter, but there is no Help for those, as they are not intended for production usage.
```

Comment:
This change absolutely makes sense, I could reproduce the failure locally using a debug build. Thanks for taking the time to fix it.